README.md

```diff
@@ -81,6 +81,29 @@ To re-index Elasticsearch based on the current database contents, run:
 bin/rails search:reindex
 ```
 
+## Troubleshooting
+
+### Flush Redis
```

Comment:
Might be worth giving a situation where this is useful and what the symptoms of the problems are.

Reply:
Thanks @kentsanggds I've added some extra info, can you remember if we had any other issues that I can add here as an example?

Reply:
I wonder if it's possible to provide an example of the error that was experienced when the redis database was out of sync?

Reply:
Yep good point, I've now added it here https://github.com/alphagov/datagovuk_publish/pull/768/files#diff-04c6e90faac2675aa89e2176d2eec7d8R122